flexmeasures/data/models/reporting/pandas_reporter.py

```diff
@@ -129,11 +134,16 @@ def _compute_report(self, **kwargs) -> list[dict[str, Any]]:
         return results
 
     def _clean_belief_series(
-        self, belief_series: tb.BeliefsSeries, belief_time: datetime
+        self,
+        belief_series: tb.BeliefsSeries,
+        belief_time: datetime | None = None,
+        belief_horizon: timedelta | None = None,
     ) -> tb.BeliefsDataFrame:
         """Create a BeliefDataFrame from a BeliefsSeries creating the necessary indexes."""
 
         belief_series = belief_series.to_frame("event_value")
+        if belief_horizon is not None:
+            belief_time = belief_series["event_start"] + belief_horizon
```

Comment:
Should be:

```
belief_time = belief_series["event_start"] + event_resolution - belief_horizon
```

Reply:
Thanks! I've included this change into https://github.com/FlexMeasures/flexmeasures/pull/1013